jax/_src/config.py

```diff
@@ -585,6 +585,22 @@ def update_thread_local_jit_state(**kw):
     )
 )
 
+jax_platforms = config.define_string_state(
+    name='jax_platforms',
+    default=None,
+    help=(
+        'Comma-separated list of platform names specifying which platforms jax '
+        'should initialize. If any of the platforms in this list is not successfully '
+        'initialized, an exception will be raised and the program will be aborted. '
+        'The first platform in the list will be the default platform. '
+        'For example, config.jax_platforms=cpu,tpu means that CPU and TPU backends '
+        'will be initialized, and the CPU backend will be used unless otherwise '
+        'specified. If TPU initialization fails, it will abort the process returning '
+        'the error message. By default, jax will try to initialize all available '
```

Comment:
Say "it will raise an exception" instead of "will abort the process returning the error message"